src/components/GetData.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+function GetAllData({search}) {
+    const [result, setResult] = useState({ });
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
    const [result, setResult] = useState();
```